components/SponsorsSection.js

```diff
@@ -91,6 +93,9 @@ function ItemSponsor ({ name, src, type }) {
             padding: 0;
           }
         }
+        .sponsor-item-gold:hover, .sponsor-item-silver:hover {
```

Comment:
The scrolling effect should be applied to the clickable element, in this case, an `a` element.

Your pull request is based on an outdated code which does not contain the links. Please pull the latest master branch, and you should see a link.